cmd/server_amd64.go

```diff
@@ -288,6 +289,28 @@ func init() {
 			}
 		}
 
+		var metricsTracker *metrics.MetricsTracker
+		if isLocalDevMode {
+			log.Printf("not sending data metrics in dev mode")
+		} else {
+			userID := "-1" // TODO DS Determine how to get user ID in GUI mode.
```

Comment:
oh yeah, this is another issue altogether! we can just use deviceID here since we don't have a concept of a user.

Reply:
Sounds good! I'm passing the device ID for both the user ID and the device ID - seems like we might as well populate both, it'll make it easier to differentiate in the web UI too.